CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ option(DISABLE_LTO_BUILD "Disable build with Link-time optimization" OFF)
 
 add_definitions(
     -DCMAKE                             # Let the source know this is a CMAKE build
+    -D__STDC_FORMAT_MACROS
```

Comment:
I'd prefer to have a comment here to remind others of what this actually does.  "Enables printf format macros for variable sized types (e.g. size_t)"